src/Atmos/Model/ref_state.jl

```diff
@@ -3,11 +3,10 @@ using DocStringExtensions
 export NoReferenceState,
     HydrostaticState,
     IsothermalProfile,
-    LinearTemperatureProfile,
     DecayingTemperatureProfile,
     DryAdiabaticProfile
 
-using CLIMAParameters.Planet: R_d, MSLP, cp_d, grav
+using CLIMAParameters.Planet: R_d, MSLP, cp_d, grav, T_min
```

Comment:
```suggestion
using CLIMAParameters.Planet: R_d, MSLP, cp_d, grav, T_min_ref
```
(And similarly throughout. It’s important to be clear that this is a reference state minimum temperature, not a general minimum temperature.)

Reply:
Ok, I don't think we've added `T_min_ref` to CLIMAParameters yet. I'll add this

Reply:
Opened [PR 20](https://github.com/CliMA/CLIMAParameters.jl/pull/20) in CLIMAParameters @tapios